catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelper.java

```diff
@@ -41,6 +42,11 @@ public InternationalStringTypeHelper() {
     InternationalStringTypeHelper(Locale locale) {
         setLocale(locale);
     }
+    
+    InternationalStringTypeHelper(Locale locale, boolean findNearestMatch) {
```

Comment:
This looks like a slightly different problem to what the original JIRA ticket describes, in that this adds new functionality where the original problem is more a bad set of test assumptions.